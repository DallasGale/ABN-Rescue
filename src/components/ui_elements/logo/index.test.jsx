/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import Logo from '.'

const props = {
  logoOnClick: () => { },
}

describe('<Logo />', () => {
  it('renders without crashing', () => {
    const component = render(<Logo {...props} />)
    expect(component).toMatchSnapshot()
  })
})
