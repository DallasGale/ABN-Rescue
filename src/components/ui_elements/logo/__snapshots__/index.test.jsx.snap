// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="padding: 10px;"
      >
        <button
          class="styled__StyledButton-sc-1feyp33-2 OdBSt"
          data-tracking="btn-click:logo"
          type="button"
        >
          <svg
            dataname="CGU_Logo_Green"
            id="CGU_Logo_Green"
            viewBox="0 0 96.23 96.24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              class="logo__StyledCls1-sc-1kziuvy-0 htTVQO"
              d="M97.33,49A48.12,48.12,0,1,1,49.21.93,48.12,48.12,0,0,1,97.33,49"
              transform="translate(-1.09 -0.93)"
            />
            <path
              class="logo__StyledCls2-sc-1kziuvy-1 dqnIPh"
              d="M48.54,52h3.62v3.43a6.58,6.58,0,0,1-2.74.55c-4.24,0-6.84-2.78-6.85-6.94s2.6-7,7.19-7A17,17,0,0,1,57.1,44V38.53a16.68,16.68,0,0,0-7.6-1.76c-8,0-12.94,5.22-12.93,12.31s5,12.29,12.31,12.28C52.6,61.36,56,60,58,57.48V46.67h-9.4Z"
              transform="translate(-1.09 -0.93)"
            />
            <path
              class="logo__StyledCls2-sc-1kziuvy-1 dqnIPh"
              d="M15.52,49.07c0,7.08,4.88,12.29,12.23,12.29a11.37,11.37,0,0,0,6.11-1.59V54.35A10.72,10.72,0,0,1,28.18,56c-4.12,0-6.65-2.79-6.65-6.94s2.53-7,6.65-7a10.63,10.63,0,0,1,5.68,1.67V38.36a11.46,11.46,0,0,0-6.11-1.59c-7.35,0-12.23,5.21-12.23,12.3"
              transform="translate(-1.09 -0.93)"
            />
            <path
              class="logo__StyledCls2-sc-1kziuvy-1 dqnIPh"
              d="M76,37.13v14.1C76,54.09,74.29,56,71.72,56s-4.23-1.92-4.23-4.78V37.13H61.7v14.1c0,6.19,4.23,10.13,10,10.13s10-3.94,10-10.12V37.13Z"
              transform="translate(-1.09 -0.93)"
            />
          </svg>
          <div
            class="styled__StyledTaglineWrapper-sc-1feyp33-1 fhcmvl"
          >
            <span
              class="styled__StyledTagline-sc-1feyp33-0 ePTdVr"
            >
               Insuring Ambition
            </span>
          </div>
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="padding: 10px;"
    >
      <button
        class="styled__StyledButton-sc-1feyp33-2 OdBSt"
        data-tracking="btn-click:logo"
        type="button"
      >
        <svg
          dataname="CGU_Logo_Green"
          id="CGU_Logo_Green"
          viewBox="0 0 96.23 96.24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            class="logo__StyledCls1-sc-1kziuvy-0 htTVQO"
            d="M97.33,49A48.12,48.12,0,1,1,49.21.93,48.12,48.12,0,0,1,97.33,49"
            transform="translate(-1.09 -0.93)"
          />
          <path
            class="logo__StyledCls2-sc-1kziuvy-1 dqnIPh"
            d="M48.54,52h3.62v3.43a6.58,6.58,0,0,1-2.74.55c-4.24,0-6.84-2.78-6.85-6.94s2.6-7,7.19-7A17,17,0,0,1,57.1,44V38.53a16.68,16.68,0,0,0-7.6-1.76c-8,0-12.94,5.22-12.93,12.31s5,12.29,12.31,12.28C52.6,61.36,56,60,58,57.48V46.67h-9.4Z"
            transform="translate(-1.09 -0.93)"
          />
          <path
            class="logo__StyledCls2-sc-1kziuvy-1 dqnIPh"
            d="M15.52,49.07c0,7.08,4.88,12.29,12.23,12.29a11.37,11.37,0,0,0,6.11-1.59V54.35A10.72,10.72,0,0,1,28.18,56c-4.12,0-6.65-2.79-6.65-6.94s2.53-7,6.65-7a10.63,10.63,0,0,1,5.68,1.67V38.36a11.46,11.46,0,0,0-6.11-1.59c-7.35,0-12.23,5.21-12.23,12.3"
            transform="translate(-1.09 -0.93)"
          />
          <path
            class="logo__StyledCls2-sc-1kziuvy-1 dqnIPh"
            d="M76,37.13v14.1C76,54.09,74.29,56,71.72,56s-4.23-1.92-4.23-4.78V37.13H61.7v14.1c0,6.19,4.23,10.13,10,10.13s10-3.94,10-10.12V37.13Z"
            transform="translate(-1.09 -0.93)"
          />
        </svg>
        <div
          class="styled__StyledTaglineWrapper-sc-1feyp33-1 fhcmvl"
        >
          <span
            class="styled__StyledTagline-sc-1feyp33-0 ePTdVr"
          >
             Insuring Ambition
          </span>
        </div>
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
